### ISSUE TITLE
Licence?

### ISSUE THREAD

**Issue by andcastillo** [dksakkos/BatchNorm#1]
Hi. I want to use this. Is this OpenSource? Could you add a licence file? Currently the official BatchNormalization layer for tensorflow has a Bug, so your implementation looks correct and I want to use it in one of the notebooks of the Deep Learning Specialization, from DeepLearning.AI
